src/server/api/modules/recipes/service/scraper.ts

```diff
@@ -81,6 +109,7 @@ function getSchemaRecipeFromNodeList(nodeList: NodeList) {
         }
       }
     }
+    logger.error(`Unable to extract recipe metadata: ${JSON.stringify(parsedNodeContent)}`);
```

Comment:
```suggestion
    logger.error({parsedNodeContent}, `Unable to extract recipe metadata`);
```

minor change to take better advantage of pino